Expand installation instructions (#79)
ansible/ansible-dev-tools: docs/installation.md

`pip install ansible-dev-tools`

> Once installation is completed, visit [Getting Started](getting-started.md) guide for more details about ADT usage.

Reviewer: ```suggestion
Once installation is completed, see the [Getting Started](getting-started.md) guide for more details about ADT usage.
```

### Latest Releases

- GitHub

>   Visit the [ADT GitHub releases page](https://github.com/ansible/ansible-dev-tools/releases) to view the latest releases. Each release includes detailed release notes outlining new features, improvements, and bug fixes.

Reviewer: ```suggestion
  To view the latest releases, see the [ADT GitHub releases page](https://github.com/ansible/ansible-dev-tools/releases). Each release includes detailed release notes outlining new features, improvements, and bug fixes.
```

- PyPI
  The [PyPI page for ADT](https://pypi.org/project/ansible-dev-tools/) provides information on the latest stable release and allows you to download specific versions of the package.

## Upgrade

> To upgrade ADT to the latest version, you can use the following pip command:

Reviewer: ```suggestion
To upgrade ADT to the latest version, use the following pip command:
```

`pip install --upgrade ansible-dev-tools`

## Downgrade

If needed, you can downgrade ADT to a specific version using the following pip command:

`pip install ansible-dev-tools==desired-version`

## Uninstallation

> If you need to uninstall ADT, you can use the following pip command:

Reviewer: ```suggestion
If you need to uninstall ADT, use the following pip command:
```
